ui/app/src/modules/Content/Dependencies/DependencySelection.tsx

```diff
@@ -27,19 +27,26 @@ import ListItemText from '@material-ui/core/ListItemText';
 import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
 import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
-import { createCheckedItems, onClickSetChecked, selectAllDeps, updateCheckedList } from '../Publish/PublishDialog';
+import { onClickSetChecked, selectAllDeps, updateCheckedList } from '../Publish/PublishDialog';
 import Button from '@material-ui/core/Button';
 import clsx from 'clsx';
 import { useSelection } from '../../../utils/hooks/useSelection';
 import { useActiveSiteId } from '../../../utils/hooks/useActiveSiteId';
+import LookupTable from '../../../models/LookupTable';
+import { createPresenceTable } from '../../../utils/array';
+import PublishingTargetIcon from '@material-ui/icons/FiberManualRecordRounded';
+import ScheduledStateIcon from '@material-ui/icons/AccessTimeRounded';
+import palette from '../../../styles/palette';
+import { useLocale } from '../../../utils/hooks/useLocale';
+import { asLocalizedDateTime } from '../../../utils/datetime';
 
-interface DependencySelectionProps<T extends BaseItem = BaseItem> {
-  items: T[];
+interface DependencySelectionProps {
+  items?: DetailedItem[] | SandboxItem[];
```

Comment:
Actually, we're only working with detailed items so this should resolve the whole situation.
```suggestion
  items?: DetailedItem[];
```